.github/workflows/ci.yml

```diff
@@ -14,18 +14,43 @@ env:
   try-runtime-uri: wss://eden-rpc.dwellir.com:443
 
 jobs:
-  test:
+  clippy:
+    runs-on: ubuntu-latest-8-cores
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          toolchain: ${{ env.nightly }}
+          target: ${{ env.target }}
+          components: clippy
+      - name: Clippy
+        uses: actions-rs/clippy-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          args: --all-features -- -A clippy::type_complexity -A clippy::identity_op -A clippy::boxed_local -D dead_code
+          toolchain: ${{ env.nightly }}
+  rustfmt:
+    runs-on: ubuntu-latest-8-cores
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          toolchain: ${{ env.nightly }}
+          target: ${{ env.target }}
+          components: rustfmt
+
+      - name: Rustfmt Check
+        uses: actions-rust-lang/rustfmt@v1
```

Comment:
I think it could be possibly more efficient for this part to be a step for the job "clippy". Though in that case you could call it "fmt-clippy".

Reply:
If the overhead is trivial, then up to you. However if you want to keep them as steps but let them both run even when one is failed, you can use `if: success() || failure()` as suggested [here](https://stackoverflow.com/a/58859404/2746280)